classes/issue/IssueDAO.inc.php

```diff
@@ -23,7 +23,7 @@ function _cacheMiss(&$cache, $id) {
 		if ($cache->getCacheId() === 'current') {
 			$issue =& $this->getCurrentIssue($id, false);
 		} else {
-			$issue =& $this->getIssueById($id, null, false);
+			$issue =& $this->getIssueByBestIssueId($id, null, false);
```

Comment:
I think this moots the following check:
https://github.com/pkp/ojs/blob/ojs-stable-2_4_6/pages/issue/IssueHandler.inc.php#L286